test/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/services/SoftwareChoicesServiceSpec.scala

```diff
@@ -51,18 +45,24 @@ class SoftwareChoicesServiceSpec extends PlaySpec with BeforeAndAfterEach {
         )
       )
 
+      val mockDataService = mock[DataService]
```

Comment:
Couldn't you declare `val mockDataService` and `val service` once at the top of the file and reuse them in each test?